README.md

```diff
@@ -63,101 +63,23 @@ Orchard allows the definition and execution of **workflows**, where each workflo
 
 Below is an example workflow that defines a number of activities to be executed in an AWS VPC environment:
 
+You can generate a an example workflow with the following command:
+
+```
+cd example/data/data
```

Comment:
I tried with the following for mustache to work
```
cd example/data
```

Reply:
thanks, it's fixed